2020-05-13 11:02:52,031 DEBUG   MainThread:27374 [wandb_config.py:_load_defaults():122] no defaults not found in config-defaults.yaml
2020-05-13 11:02:52,037 DEBUG   MainThread:27374 [meta.py:setup():100] code probe starting
2020-05-13 11:02:52,039 DEBUG   MainThread:27374 [meta.py:_setup_code_git():49] probe for git information
2020-05-13 11:02:52,059 DEBUG   MainThread:27374 [meta.py:_setup_code_program():58] save program starting
2020-05-13 11:02:52,059 DEBUG   MainThread:27374 [meta.py:_setup_code_program():60] save program starting: /home/datta/lab/HPE3D/experiments/../src/train.py
2020-05-13 11:02:52,060 DEBUG   MainThread:27374 [meta.py:_setup_code_program():68] save program saved: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/code/src/train.py
2020-05-13 11:02:52,060 DEBUG   MainThread:27374 [meta.py:_setup_code_program():70] save program
2020-05-13 11:02:52,060 DEBUG   MainThread:27374 [meta.py:setup():122] code probe done
2020-05-13 11:02:52,066 DEBUG   MainThread:27374 [run_manager.py:__init__():546] Initialized sync for hpe/2qo1l6q9
2020-05-13 11:02:52,070 INFO    MainThread:27374 [run_manager.py:wrap_existing_process():1148] wrapping existing process 27351
2020-05-13 11:02:52,070 WARNING MainThread:27374 [io_wrap.py:register():104] SIGWINCH handler was not None: <Handlers.SIG_DFL: 0>
2020-05-13 11:02:52,125 INFO    MainThread:27374 [run_manager.py:init_run():927] system metrics and metadata threads started
2020-05-13 11:02:52,126 INFO    MainThread:27374 [run_manager.py:init_run():966] upserting run before process can begin, waiting at most 10 seconds
2020-05-13 11:02:52,480 INFO    Thread-14 :27374 [run_manager.py:_upsert_run():1051] saving patches
2020-05-13 11:02:52,602 INFO    Thread-14 :27374 [run_manager.py:_upsert_run():1055] saving pip packages
2020-05-13 11:02:52,604 INFO    Thread-14 :27374 [run_manager.py:_upsert_run():1057] initializing streaming files api
2020-05-13 11:02:52,605 INFO    Thread-14 :27374 [run_manager.py:_upsert_run():1064] unblocking file change observer, beginning sync with W&B servers
2020-05-13 11:02:52,605 INFO    MainThread:27374 [run_manager.py:wrap_existing_process():1165] informing user process we are ready to proceed
2020-05-13 11:02:52,612 INFO    MainThread:27374 [run_manager.py:_sync_etc():1272] entering loop for messages from user process
2020-05-13 11:02:52,614 INFO    MainThread:27374 [run_manager.py:_sync_etc():1295] received message from user process: {"save_policy": {"glob": "events.out.tfevents.1589360571.Datta.27351.0", "policy": "live"}}
2020-05-13 11:02:53,041 INFO    Thread-3  :27374 [run_manager.py:_on_file_modified():691] file/dir modified: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/config.yaml
2020-05-13 11:02:53,153 INFO    MainThread:27374 [run_manager.py:_sync_etc():1328] process received interrupt signal, shutting down
2020-05-13 11:02:53,155 INFO    MainThread:27374 [run_manager.py:_sync_etc():1381] closing log streams and sending exitcode to W&B
2020-05-13 11:02:53,156 INFO    MainThread:27374 [run_manager.py:shutdown():1071] shutting down system stats and metadata service
2020-05-13 11:02:53,278 INFO    Thread-3  :27374 [run_manager.py:_on_file_created():680] file/dir created: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/diff.patch
2020-05-13 11:02:53,278 INFO    Thread-3  :27374 [run_manager.py:_on_file_created():680] file/dir created: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/events.out.tfevents.1589360571.Datta.27351.0
2020-05-13 11:02:53,279 INFO    Thread-3  :27374 [run_manager.py:_on_file_created():680] file/dir created: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/wandb-metadata.json
2020-05-13 11:02:53,279 INFO    Thread-3  :27374 [run_manager.py:_on_file_created():680] file/dir created: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/code/src/train.py
2020-05-13 11:02:53,279 INFO    Thread-3  :27374 [run_manager.py:_on_file_created():680] file/dir created: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/output.log
2020-05-13 11:02:53,280 INFO    Thread-3  :27374 [run_manager.py:_on_file_created():680] file/dir created: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/requirements.txt
2020-05-13 11:02:53,280 INFO    Thread-3  :27374 [run_manager.py:_on_file_created():680] file/dir created: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/code
2020-05-13 11:02:53,280 INFO    Thread-3  :27374 [run_manager.py:_on_file_created():680] file/dir created: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/code/src
2020-05-13 11:02:54,040 INFO    Thread-3  :27374 [run_manager.py:_on_file_modified():691] file/dir modified: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/output.log
2020-05-13 11:02:54,040 INFO    Thread-3  :27374 [run_manager.py:_on_file_modified():691] file/dir modified: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/wandb-history.jsonl
2020-05-13 11:02:54,041 INFO    Thread-3  :27374 [run_manager.py:_on_file_created():680] file/dir created: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/wandb-summary.json
2020-05-13 11:02:54,044 INFO    Thread-3  :27374 [run_manager.py:_on_file_created():680] file/dir created: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/wandb-events.jsonl
2020-05-13 11:02:55,042 INFO    Thread-3  :27374 [run_manager.py:_on_file_modified():691] file/dir modified: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/wandb-metadata.json
Thread-3  :27374 [2qo1l6q9:run_manager.py:_on_file_created():680] file/dir created: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/wandb-events.jsonl
2020-05-13 11:02:55,042 INFO    Thread-3  :27374 [2qo1l6q9:run_manager.py:_on_file_modified():691] file/dir modified: /home/datta/lab/HPE3D/experiments/wandb/run-20200513_090251-2qo1l6q9/wandb-metadata.json
